Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -243,6 +243,10 @@ namespace Babylon
             }
             m_isFrameScheduled = true;
 
+            m_graphicsImpl.GetAfterRenderTask().then(arcana::inline_scheduler, arcana::cancellation::none(), [this] {
+                m_engineImpl->ScheduleRender();
+            });
+
```

Comment:
nit: maybe move this after GetBeforeRenderTask so that Before is before and After is after?

Reply:
Could do, but this ordering was intended to communicate the logical progression, specifically how the after render even precipitates the before render event on the next frame. I'm not sure it gets the message across, though.